Admin.js

```diff
@@ -11,7 +11,7 @@
 
 var Admin = {
```

Comment:
- eslint: Expected dot to be on same line as object.
- eslint: Strings must use doublequote.